tests/test_utils.py

```diff
@@ -582,7 +582,8 @@ def test_clone_git_repo_commit_failure(tmpdir, commit, branch, depth):
     with pytest.raises(OsbsException) as exc:
         clone_git_repo(TEST_DOCKERFILE_GIT, tmpdir_path, commit=commit, retry_times=1,
                        branch=branch, depth=depth)
-    assert 'cannot find commit' in exc.value.message
+    assert re.match(r'^Commit \S+ is not reachable in branch \S+, '
+                    r'reason: (\S+\s?)*$', str(exc.value))
```

Comment:
Instead of using regex matching, could the error message just be formatted with the expected commit and branch?